tests/Common/ComponentsTest.php

```diff
@@ -464,31 +515,50 @@ public function testComponentConfigRestrictionsForReadOnlyUser()
         $this->assertSame($components, $componentsForAdmin->listComponents());
     }
 
-    public function testConfigurationNameShouldBeRequired()
+    /**
+     * @dataProvider componentsProvider
+     */
+    public function testConfigurationNameShouldBeRequired(callable $getClient)
     {
+        /** @var Client $client */
+        $client = $getClient($this);
+        $components = new \Keboola\StorageApi\Components($client);
```

Comment:
na co jsou tady ty $components? ani v constructoru to nedela zadnou logiku (tak nejak vsude)

Reply:
ha, to je pozustatek z predchozi iterace. Dik